code/Commenting.php

```diff
@@ -13,12 +13,18 @@
  */
 
 class Commenting {
-	
+
 	/**
 	 * @var array map of enabled {@link DataObject} and related configuration
 	 */
 	private static $enabled_classes = array();
-	
+
+	/**
+	 * @config
+	 * @var bool Whether to enable commenting on SiteTree objects by default
+	 */
+	private static $sitetree_comments = true;
+
```

Comment:
> Whether to enable commenting on SiteTree objects by default

We already have Commenting::has_commenting() which I would have thought is false if SiteTree commenting is disabled.